project/components/crud.py

```diff
@@ -66,6 +66,11 @@ async def __aexit__(self, *args: Any) -> None:
 
         return None
 
+    @property
+    def select_query(self) -> select:
```

Comment:
Here it will be an instance of `Select` class from `from sqlalchemy.sql import Select`.
```suggestion
    def select_query(self) -> Select:
```